src/stdx.zig

```diff
@@ -266,6 +266,29 @@ pub const log = if (builtin.is_test)
 else
     std.log;
 
+/// An alternative to the default logFn from `std.log`, which prepends a UTC timestamp.
+pub fn log_with_timestamp(
```

Comment:
When bringing things into `stdx`, prefer rewriting them to match existing TigerStyle rules (eg `log_override`) over trying to keep them closer to the original source

Reply:
Sorry, I meant to highlight the entire block - I meant things like `const prefix2` and `bw`.

Reply:
(The name `log_with_timestamp` is good)